Cargo.toml

```diff
@@ -35,7 +35,7 @@ ic-response-verification = "2.6"
 ic-stable-structures = "0.6"
 ic-verifiable-credentials = {git = "https://github.com/dfinity/verifiable-credentials-sdk", rev = "b74c746ea5361af3da207a2c957be4a951f7a72c"}
 ic-canister-sig-creation = "1.1"
-pocket-ic = "4.0"
+pocket-ic = "5.0"
```

Comment:
This doesn't match the version in downloading?

Reply:
Client library version and binary are out of sync and have different versions: https://docs.google.com/document/d/1VYmHUTjrgbzRHtsAyRrI5cj-gWGs7ktTnutPvUMJioU/edit#heading=h.5wf28dvt742x

Reply:
🙈